### PR TITLE
make tests that involve GDrive more robust

### LINKED ISSUE #4425: Requests on Google drive lead to "HTTP Error 429: Too Many Requests" errors
### 🐛 Describe the bug

I started receiving unrelated errors on test `test_download_url_dispatch_download_from_google_drive`:
```
  File "/root/project/test/test_internet.py", line 46, in test_download_url_dispatch_download_from_google_drive
    utils.download_url(url, tmpdir, filename, md5)
  File "/root/project/torchvision/datasets/utils.py", line 130, in download_url
    url = _get_redirect_url(url, max_hops=max_redirect_hops)
  File "/root/project/torchvision/datasets/utils.py", line 78, in _get_redirect_url
    with urllib.request.urlopen(urllib.request.Request(url, headers=headers)) as response:
  File "/root/project/env/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/root/project/env/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 429: Too Many Requests
```

Example: https://app.circleci.com/pipelines/github/pytorch/vision/10460/workflows/7ab4bef5-1768-417f-8bbd-c92d2451fcd8/jobs/794189

### Versions

Latest master https://github.com/pytorch/vision/commit/3380c001c0d5b31ac8558599656404cf4e28ecd1

cc @pmeier

### PR DESCRIPTION
**Pull request by pmeier**
Fixes #4425. Since the error is gone now, I can't debug it further. As explained in the issue, my guess is that the file we were testing against was also requested by a lot of other users and we were hitting a rate limit. With this change we are testing a dedicated dummy file that should not have these issues.

cc @pmeier

### CODE REVIEW

File: test/test_datasets_utils.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def test_check_integrity(self):
         assert not utils.check_integrity(nonexisting_fpath)
 
     def test_get_google_drive_file_id(self):
-        url = "https://drive.google.com/file/d/1hbzc_P1FuxMkcabkgn9ZKinBwW683j45/view"
+        url = "https://drive.google.com/file/d/1GO-BHUYRuvzr1Gtp2_fqXRsr9TIeYbhV/view"
         expected = "1hbzc_P1FuxMkcabkgn9ZKinBwW683j45"
```
Comment (prabhat00155) [inline test/test_datasets_utils.py:80]: Will `expected` still be the same?

**Review by prabhat00155 — COMMENTED** [review]
Thanks @pmeier for the PR.

File: test/test_datasets_utils.py (hunk 1 above)
Comment (pmeier) [reply test/test_datasets_utils.py:80]: Oops, no of course not. Thanks!

Comment (prabhat00155) [reply test/test_datasets_utils.py:80]: Same for the other test as well.

File: test/test_internet.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def test_download_url_dont_exist(self, tmpdir):
             utils.download_url(url, tmpdir)
 
     def test_download_url_dispatch_download_from_google_drive(self, mocker, tmpdir):
-        url = "https://drive.google.com/file/d/1hbzc_P1FuxMkcabkgn9ZKinBwW683j45/view"
+        url = "https://drive.google.com/file/d/1GO-BHUYRuvzr1Gtp2_fqXRsr9TIeYbhV/view"
 
         id = "1hbzc_P1FuxMkcabkgn9ZKinBwW683j45"
```
Comment (prabhat00155) [inline test/test_internet.py:41]: Perhaps, you need to update `id` here.

**Review by prabhat00155 — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**